stdlib/asyncio/taskgroups.pyi

```diff
@@ -0,0 +1,15 @@
+# This only exists in 3.11+. See VERSIONS.
+
+from types import TracebackType
+from typing import Any, Coroutine, Generator, TypeVar
+
+from .tasks import Task
+
+_T = TypeVar("_T")
+
+class TaskGroup:
+    def __init__(self, name: str | None = ...) -> None: ...
+    def get_name(self) -> str: ...
+    async def __aenter__(self) -> TaskGroup: ...
+    async def __aexit__(self, et: type[BaseException] | None, exc: BaseException | None, tb: TracebackType | None) -> None: ...
+    def create_task(self, coro: Generator[Any, None, _T] | Coroutine[Any, Any, _T], *, name: str | None = ...) -> Task[_T]: ...
```

Comment:
Just noticed that the `name` argument doesn't seem to exist on cpython's `main` branch:

```suggestion
    def create_task(self, coro: Generator[Any, None, _T] | Coroutine[Any, Any, _T]) -> Task[_T]: ...
```

Reply:
Good catch! Fixed.